[Vagrantfile] Synced folder modification

gbrennon opened the pull request:
-synced_folder is pointing to $HOME/ng-box/projects and will create the
dir, if it doesn't exist.
-Server port changed from 8080 to 8180 because 8080 is Jenkins default
port.

gbrennon committed:
[Vagrantfile]Synced folder and server port modif

-synced_folder is pointing to $HOME/ng-box/projects and will create the
dir, if it doesn't exist.
-Server port changed from 8080 to 8180 because 8080 is Jenkins default
port.

willmendesneto commented on Vagrantfile:
```diff
@@ -52,7 +52,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # the path on the guest to mount the folder. And the optional third
   # argument is a set of non-required options.
-  config.vm.synced_folder "/home/willmendesneto/projects/ng-box/projects", "/vagrant_data"
+  config.vm.synced_folder ENV['HOME'] + "/ng-box/projects", "/vagrant_data", create: true
```
Great @gbrennon, this code is really incorrect. Remove the user is correct, but use `ENV['HOME']`, not. This code should be updated for the developer that will use the vagrant box. This code works on in *unix system operation, but doesn't works in Windows environment. It's good modify this code for `../vagrant-frontend-mobile-box/projects`, for example

gbrennon committed:
Revert "[Vagrantfile]Synced folder and server port modif"

-Reverting this commit to make 2 commits. 1 for Jenkins port,
and the other for the synced_folder issue

This reverts commit 6bdab67d3d7d955a981bb82ffc53596a4e52aeff.

gbrennon committed:
[Vagrantfile]synced_folder now have a HOME ref

-synced_folder points to a HOME reference and create, recursively,
needed folders

gbrennon committed:
[Vagrantfile]synced_folder now works on non *nyx

-synced_folder reference changed from HOME folder to
'../vagrant-frontend-mobile-box'

gbrennon committed:
[Vagrantfile]Correction on synced_folder dir

willmendesneto commented:
Thanks for contribute @gbrennon ! Great job!
